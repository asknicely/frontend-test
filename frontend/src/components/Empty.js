import React from 'react'

const Empty = () => {
    return (
        <tr>
            <td colSpan="5">No todos found ...</td>
        </tr>
    )
}

export default Empty