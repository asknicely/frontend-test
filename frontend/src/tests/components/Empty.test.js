import React from 'react';
import { shallow } from 'enzyme';
import Empty from '../../components/Empty';

test('should render Empty correctly', () => {
	const wrapper = shallow(
		<Empty />
	);
	expect(wrapper).toMatchSnapshot();
});
