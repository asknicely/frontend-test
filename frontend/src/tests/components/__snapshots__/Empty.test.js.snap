// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Empty correctly 1`] = `
<tr>
  <td
    colSpan="5"
  >
    No todos found ...
  </td>
</tr>
`;
